packages/safe-service-client/tests/endpoint.test.ts

```diff
@@ -33,210 +33,14 @@ describe('Endpoint tests', () => {
-      const signature = await signer.signMessage(data)
-      await chai
-        .expect(serviceSdk.removeAllSafeDelegates(safeAddress, signer))
-        .to.be.eventually.deep.equals({ data: { success: true } })
-      chai.expect(fetchData).to.have.been.calledWith({
-        url: `${getTxServiceBaseUrl(txServiceBaseUrl)}/safes/${safeAddress}/delegates/`,
-        method: 'delete',
-        body: { signature }
-      })
-    })
-
-    it('removeSafeDelegate', async () => {
-      const provider = getDefaultProvider(config.JSON_RPC)
-      const signer = new Wallet(
-        '0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d', // A Safe owner
-        provider
-      )
-      const delegate = '0x22d491Bde2303f2f43325b2108D26f1eAbA1e32b'
-      const delegateConfig: SafeDelegateDeleteConfig = {
-        safe: safeAddress,
-        delegate,
-        signer
-      }
-      const totp = Math.floor(Date.now() / 1000 / 3600)
-      const data = delegate + totp
-      const signature = await signer.signMessage(data)
-      await chai
-        .expect(serviceSdk.removeSafeDelegate(delegateConfig))
-        .to.be.eventually.deep.equals({ data: { success: true } })
-      chai.expect(fetchData).to.have.been.calledWith({
-        url: `${getTxServiceBaseUrl(txServiceBaseUrl)}/safes/${safeAddress}/delegates/${
-          delegateConfig.delegate
-        }`,
-        method: 'delete',
-        body: {
-          safe: delegateConfig.safe,
-          delegate: delegateConfig.delegate,
-          signature
-        }
-      })
-    })
-
-    it('getSafeCreationInfo', async () => {
-      await chai
-        .expect(serviceSdk.getSafeCreationInfo(safeAddress))
-        .to.be.eventually.deep.equals({ data: { success: true } })
-      chai.expect(fetchData).to.have.been.calledWith({
-        url: `${getTxServiceBaseUrl(txServiceBaseUrl)}/safes/${safeAddress}/creation/`,
-        method: 'get'
-      })
-    })
-
-    it('estimateSafeTransaction', async () => {
-      const safeTransaction: SafeMultisigTransactionEstimate = {
-        to: ownerAddress,
-        value: '0',
-        data: '0x',
-        operation: 0
-      }
-      await chai
-        .expect(serviceSdk.estimateSafeTransaction(safeAddress, safeTransaction))
-        .to.be.eventually.deep.equals({ data: { success: true } })
-      chai.expect(fetchData).to.have.been.calledWith({
-        url: `${getTxServiceBaseUrl(
-          txServiceBaseUrl
-        )}/safes/${safeAddress}/multisig-transactions/estimations/`,
-        method: 'post',
-        body: safeTransaction
-      })
-    })
 
     it('proposeTransaction', async () => {
       const safeTxData: SafeTransactionDataPartial = {
         to: '0xa33d2495760462018275994d85117600bd58221e',
         data: '0x',
         value: '123456789',
         operation: 1,
-        safeTxGas: 123,
+        safeTxGas: 0,
         baseGas: 0,
         gasPrice: 0,
         gasToken: '0x0000000000000000000000000000000000000000',
```

Comment:
Do you use the a `ZERO_ADDRESS` often? Might be worthwhile creating a constant for it.

Reply:
It is only used twice in this test file